app/_components/commodities/Form.tsx

```diff
@@ -114,108 +116,120 @@ const Form: React.FC<FormProps> = ({ onSubmitHandler, isLoading }) => {
-                <Radio
-                  colorScheme="orange"
-                  key={index}
-                  value={value.toLowerCase()}
-                  borderColor={selectColor(isDark, 'border')}
-                  {...register('maxLandingPadSize', {
-                    required: true,
-                  })}
-                >
-                  {value}
-                </Radio>
-              ))}
-              <FormErrorMessage marginY="auto">
-                {errors.maxLandingPadSize && 'Select a landing pad size'}
-              </FormErrorMessage>
+              <CheckboxGroup colorScheme="gray">
+                {checkboxValues.map((checkbox, index) => (
+                  <Checkbox
+                    colorScheme="orange"
+                    key={index}
+                    {...register(`${checkbox.value}` as keyof SubmitProps)}
+                    borderColor={selectColor(isDark, 'border')}
+                  >
+                    {checkbox.name}
+                  </Checkbox>
+                ))}
+              </CheckboxGroup>
             </Stack>
-          </RadioGroup>
-        </FormControl>
-        <FormControl isInvalid={!!errors.minSupply || !!errors.minDemand}>
-          <FormLabel>I am looking to:</FormLabel>
-          <Stack spacing={8} direction="row" margin={8} flexWrap="wrap">
-            <ButtonGroup size="md" isAttached>
+          </FormControl>
+          <FormControl
+            width="50%"
+            isInvalid={
+              !!(errors.maxLandingPadSize && errors.maxLandingPadSize.message)
+            }
+          >
+            <FormLabel>Landing Pad Size</FormLabel>
+            <RadioGroup>
+              <Stack
+                borderWidth="1px"
+                borderRadius="md"
+                borderColor={selectColor(isDark, 'border')}
+                padding={3}
+                spacing={5}
+                direction="row"
+              >
+                {['Small', 'Medium', 'Large'].map((value, index) => (
+                  <Radio
+                    colorScheme="orange"
+                    key={index}
+                    value={value.toLowerCase()}
+                    borderColor={selectColor(isDark, 'border')}
+                    {...register('maxLandingPadSize', {
+                      required: true,
+                    })}
+                  >
+                    {value}
+                  </Radio>
+                ))}
+                <FormErrorMessage marginY="auto">
+                  {errors.maxLandingPadSize && 'Select a landing pad size'}
+                </FormErrorMessage>
+              </Stack>
+            </RadioGroup>
+          </FormControl>
+        </HStack>
+        <HStack spacing={4} marginTop={4} width="100%">
+          <FormControl
+            width="20%"
+            isInvalid={!!errors.minSupply || !!errors.minDemand}
+          >
+            <FormLabel>I am looking to:</FormLabel>
```

Comment:
Minor grammar nazism, but that's me.

We have a choice here, the language can either be
'I am looking to buy/sell' or 'I am buying/selling'

'I am looking to buying/selling' doesn't work

Reply:
Changed one thing and left the other. Will fix this.

Reply:
Fixed in next commit